<ui version="4.0" >
 <author>SlowTree</author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>UICelere</class>
 <widget class="QWidget" name="UICelere" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>4</hsizetype>
    <vsizetype>4</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>400</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxResults" >
   <property name="geometry" >
    <rect>
     <x>5</x>
     <y>120</y>
     <width>380</width>
     <height>320</height>
    </rect>
   </property>
   <property name="title" >
    <string>Risultati</string>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QTreeWidget" name="listViewResponse" >
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
      <property name="itemsExpandable" >
       <bool>false</bool>
      </property>
      <property name="sortingEnabled" >
       <bool>true</bool>
      </property>
      <column>
       <property name="text" >
        <string>Data</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Stato Spedizione</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Agenzia/Filiale SDA</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxData" >
   <property name="geometry" >
    <rect>
     <x>5</x>
     <y>0</y>
     <width>380</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title" >
    <string>Dati</string>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="1" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>81</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2" >
     <widget class="QLineEdit" name="lineEditCode" >
      <property name="maxLength" >
       <number>7</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3" >
     <widget class="QPushButton" name="pushButtonSend" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="cursor" >
       <cursor>13</cursor>
      </property>
      <property name="text" >
       <string>&amp;Controlla</string>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3" >
     <widget class="QPushButton" name="pushButtonCancel" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="cursor" >
       <cursor>13</cursor>
      </property>
      <property name="text" >
       <string>&amp;Annulla</string>
      </property>
      <property name="icon" >
       <iconset/>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Numero</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>lineEditCode</tabstop>
  <tabstop>pushButtonSend</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>listViewResponse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
